lhotse/features/kaldi/layers.py

```diff
@@ -506,25 +510,12 @@ def __init__(
         )
 
     def forward(self, x):
-        x_strided = self.wav2win(x)
-        if self.use_energy:
-            x_strided, log_e = x_strided
+        x_strided, log_e = self.wav2win(x)
 
         X = _rfft(x_strided)
         pow_spec = self._to_spec(X)
 
-        try:
-            from torch.cuda.amp import autocast
-        except ImportError:
-            warnings.warn(
-                "Could not import torch.cuda.amp.autocast -- "
-                "when using mixed precision with another package such as apex, "
-                "you might experience numerical stability issues."
-            )
-            pow_spec = torch.matmul(pow_spec.float(), self._fb.float())
-        else:
-            with autocast(enabled=False):
-                pow_spec = torch.matmul(pow_spec.float(), self._fb.float())
+        pow_spec = torch.matmul(pow_spec.float(), self._fb.float())
```

Comment:
@jesus-villalba do you think this change could be dangerous with fp16?

Reply:
That multiplication only gave me trouble when the signal is in (-2^15, 2^15). If the signal is in (-1,1) I think fp16 worked fine

Reply:
Hm so can I leave it as is @pzelasko ?

Reply:
Yeah